core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -274,6 +281,43 @@ public static String jsonLogicalPlan(
                 anonymizer).toJson();
     }
 
+    public static String anonymizedJsonRenderedPlan(
```

Comment:
anonymizedJsonRenderedPlan -> jsonDistrributedPlan with Anonymizer as input parameter.
You can use this to implement distributed json explain in a prep commit